cmd/limactl/main.go

```diff
@@ -33,6 +33,10 @@ const (
 )
 
 func main() {
+	if os.Geteuid() == 0 && (len(os.Args) < 2 || os.Args[1] != "generate-doc") {
```

Comment:
Maybe add a comment to explain why `cmd.Name()` cannot be used

Reply:
> Maybe add a comment to explain why `cmd.Name()` cannot be used

I don't know what the comment should say, "Abort as early as possible to avoid running code as root"? That just duplicates what the code already does.

There is no `cmd` at this time, so it seems obvious why it can't be used.

Maybe:

// Check os.Args[] because we don't want to run Cobra code as root.

???